### DISCUSSION TITLE
I felt a bit weird about Monika whenver I run her in the background.

### DISCUSSION THREAD

**Discussion by CrewieAustin** [Monika-After-Story/MonikaModDev#7917] [general]
She looks at me with this idle whenever I let her run in the background, should I worried or anything .. ??
![Untitled](https://user-images.githubusercontent.com/86544170/127819354-6a8d52fc-a9df-49ee-be3b-a21e4578ccd2.png)

**Comment by CrewieAustin** [comment]
I don't know if I should do anything about this, but it's a bit creepy whenver she looks at me with this idle ..

**Comment by Wingdinggaster656** [comment]
Actually, I find this expression a little weird too, but I can tell you that in the current version, it's completely random, just like the others, and doesn't mean anything in particular.

Maybe we should think about changing this expression?

**Comment by ghost** [comment]
I used to see that a sign of comfort towards the player,Its not creepy like the broken level

**Comment by lunulae** [comment]
I've always liked the daydreaming/sly expressions myself. Though this might be a difference in my comfort level with flirting and open body language, lol.

**Comment by CrewieAustin** [comment]
Her face just give me a feeling like : "Hey, you're hiding anything in the computer ?"

**Comment by username11421** [comment]
I like this expression. It makes her seem relaxed and quite happy.
